samples/net/wpan_serial/app.overlay

```diff
@@ -0,0 +1,14 @@
+/*
+ * Copyright (c) 2021 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+/ {
+	usbd0 {
```

Comment:
Does this really work? I thought aliases could not be used this way in your testing. The same question applies to other app.overlay files in this PR that do the same thing.

Reply:
That works, what does not work is to define cdc_acm_uart0 node as a child of a node using reference and enable it here.